llm/memory_test.go

```diff
@@ -20,10 +19,9 @@ func TestEstimateGPULayers(t *testing.T) {
 	f, err := os.CreateTemp(t.TempDir(), modelName)
 	require.NoError(t, err)
 	defer f.Close()
-	gguf := NewGGUFV3(binary.LittleEndian)
 	inputLayerCount := 5
 
-	tensors := []Tensor{
+	tensors := []*Tensor{
```

Comment:
Is there a benefit to making this a pointer (it seems that this only contains a few values)

Reply:
It was original changed to a pointer because Offset is being modified but that isn't necessary anymore so i reverted